spec/support/shared_examples/year_validations.rb

```diff
@@ -0,0 +1,37 @@
+RSpec.shared_examples 'year validations' do |year_field, validations|
+  include DateAndYearConcerns
+
+  let(year_field.to_sym) { year }
+
+  describe 'when invalid' do
```

Comment:
Might be superfluous, but maybe we should add a when valid describe block as well as some of those are custom validations

Reply:
Good idae, but maybe in the year validator specs rather than the shared examples, as then I can easily verify the validity of the object without relying on additional data setup.